proposals/0192-non-exhaustive-enums.md

```diff
@@ -79,7 +89,7 @@ case .thoughtItWasDueNextWeek:
 }
 ```
 
-In Swift 5, this would be an error. To maintain source compatibility, this would only produce a warning in Swift 4 mode. The Swift 4 program will trap at run time if an unknown enum case is actually encountered.
+Failure to do so will produce a warning in Swift 5. A program will trap at run time if an unknown enum case is actually encountered.
```

Comment:
Likewise, same problem. I thought your proposal fixed the previous iteration of this by making non-frozen enums a lot less common in practice.  Is there a third category of resilient enum being introduced here?

Reply:
Right, maybe that's my confusion.  There are actually three things going on here:

1.  There are explicitly @frozen enums.
2. There are non-frozen Swift enums, which are the default.
3. There is some form of resilient enum which "only affects C enums and enums in the standard library and overlays today."

Given that, it makes sense that this would be a warning, not an error.  Sorry for the confusion!

Reply:
Yep. If/when we standardize the 'frozen' attribute we can get another chance at clearing up this terminology.